src/classes/AFFL_MultiRecordTypeMapper.cls

```diff
@@ -247,20 +249,27 @@ public with sharing class AFFL_MultiRecordTypeMapper {
     /*******************************************************************************************************
     * @description Returns valid contact primary affiliation fields from mappings
     * @param cf List of fields.
-    * @param primaryAfflFieldLabels List of field labels.
+    * @param primaryAfflFieldsFromSettings List of field labels/API Names from settings.
     * @return List<SObjectField> List of valid contact primary affiliation fields from mappings.
     ********************************************************************************************************/
-    private List<String> getValidAfflFields(List<SObjectField> cf, List<String> primaryAfflFieldLabels) {
-        Set<String> contactFieldStrings = new Set<String>();
+    @testVisible
+    private List<String> getValidAfflFields(List<SObjectField> contactFields, List<String> primaryAfflFieldsFromSettings) {
+
+        Set<String> contactFieldLabelStrings = new Set<String>();
```

Comment:
We do have class variables doing the samething above. See contactLabelNames.

Reply:
This was an old logic, but updated it to use to public variables.